app/src/main/java/org/coepi/android/domain/symptomflow/SymptomInputs.kt

```diff
@@ -8,15 +8,16 @@ import java.io.Serializable
 
 enum class SymptomId {
     COUGH, BREATHLESSNESS, FEVER, MUSCLE_ACHES, LOSS_SMELL_OR_TASTE, DIARRHEA,
-    RUNNY_NOSE, OTHER, NONE
+    RUNNY_NOSE, OTHER, NONE, EARLIESTSYMPTOM
```

Comment:
The earliest symptom date is just an input / step. It should not be a symptom id.